src/hooks/ykfde

```diff
@@ -72,6 +73,7 @@ EOF
   [ -z "$YKFDE_LUKS_DEV" ] && YKFDE_LUKS_DEV="/dev/disk/by-uuid/$YKFDE_DISK_UUID"
   [ "$YKFDE_CRYPTSETUP_TRIALS" -lt 1 ] && YKFDE_CRYPTSETUP_TRIALS="5"
   [ "$YKFDE_CHALLENGE_SLOT" -lt 1 ] || [ "$YKFDE_CHALLENGE_SLOT" -gt 2 ] && YKFDE_CHALLENGE_SLOT="2"
+  [ "$YKFDE_SKIP_PASSWORD_PROMPT" -ne 1 ] && YKFDE_SKIP_PASSWORD_PROMPT="0"
```

Comment:
we accept any non-empty value for other true/false options so strictly expecting `1` would be rather inconsequential.